au/code/au/quantity_point.hh

```diff
@@ -370,6 +370,15 @@ constexpr auto operator-(QuantityPoint<U1, R1> p1, QuantityPoint<U2, R2> p2) {
     return detail::using_common_point_unit(p1, p2, detail::minus);
 }
 
+// Spaceship operator provides C++20 compatibility.
+#if __cplusplus >= 202002L
```

Comment:
Did you consider `#if defined(__cpp_impl_three_way_comparison) && __cpp_impl_three_way_comparison >= 201907L`? I think it's more self describing and may improve support for toolchains that have only partial C++20 support

Reply:
I didn't think super deeply about it.  Feature test macros are pretty new to me, so I went with what seemed simpler.  If this gives us all the same benefits, but supports even more toolchains, we could migrate to it --- apart from being more clunky and verbose, it doesn't seem any worse than the other feature test macro.

Reply:
:+1: I'll put up a PR and we can evaluate it in the reviews there

Reply:
Opened https://github.com/aurora-opensource/au/pull/358, let's evaluate it over there